### PR TITLE
feature(PCO::URL): add methods for encrypting and decrypting params

### PR DESCRIPTION
**Pull request by geolessel**
Open for discussion

### CODE REVIEW

File: lib/pco/url.rb [hunk 1]
```diff
@@ -1,14 +1,28 @@
 require "pco/url/version"
 require 'uri'
+require 'base64'
 
 module PCO
   class URL
 
+    SALT = 'These_pretzels_are_making_me_thirsty'
```
Comment (danott) [inline lib/pco/url.rb]: Salty indeed. :smile:

**Comment by danott** [comment]
Is the intent to send arbitrary params, encoded?

**Comment by geolessel** [comment]
@danott Yes. This is just a concept I wanted feedback on. The implementation here obviously isn't really secure and the SALT doesn't really do anything. I dm'ed with James regarding the idea of this and he has an idea for moving forward with this.

**Comment by bensie** [comment]
@danott Not just encoded, encrypted. I'm working this up now.

**Comment by danott** [comment]
Could we solve this with Rails `GlobalID` or `MessageVerifier`? Could be interesting to standardize how we send these kinda of short messages between apps.

**Comment by geolessel** [comment]
@danott I was heading towards `MessageVerifier` but Services is not on Rails 4.1 yet. My intention was to make this sort of thing a standard which is why I thought putting it into pco-url would be a good idea.

**Comment by danott** [comment]
I agree. `PCO::URL` feels like the right place

**Comment by bensie** [comment]
Closing in favor of #5

**Pull request closed without merging**